src/features/MoviePage/Banner/styled.js

```diff
@@ -3,30 +3,29 @@ import styled from "styled-components";
 export const PosterContainer = styled.div`
   margin: 0 auto;
   background-color: ${({ theme }) => theme.color.black};
+  position: relative;
+  z-index: -3;
 `;
+
 export const ImagesWrapper = styled.div`
   max-width: 1356px;
-  position: relative;
   max-height: 770px;
+  position: relative;
   margin: 0 auto;
-
-  @media (max-width: ${({ theme }) => theme.breakpoint.narrow}) {
-    width: 90%;
-  }
 `;
 
-export const Shadow = styled.img`
+export const StyledPoster = styled.div`
   max-width: 1356px;
-  height: 100%;
-  z-index: 1;
-  position: absolute;
-  top: 0;
-  left: 0;
+  max-height: 770px;
+  box-shadow: inset 0 0 100px 100px rgba(0, 0, 0);
```

Comment:
What do you think about small changing of box-shadow from this to 
box-shadow: inset 0px 0px **150px 140px** rgba(0, 0, 0);
At least for bigger displays.